docs/guidance/example.md

```diff
@@ -44,21 +75,21 @@ This section provides contextual information about the project, including:
 .. jsoninclude:: ../examples/example.json
    :jsonpointer: /projects/0/locations
    :expand:  geometry, gazetteer, address
-   :title: location 1
+   :title: location1
```

Comment:
Is there a reason not just to use 'location'?

I suggest updating the previous paragraph to read:

> In this example, one location is given: a motorway junction, using a point location, a gazetteer entry and an address.